src/validate.rs

```diff
@@ -185,15 +185,6 @@ fn validate_subteam_of(data: &Data, errors: &mut Vec<String>) {
                 );
             };
 
-            if !matches!(team.kind(), TeamKind::Team) && parent.subteam_of().is_some() {
-                bail!(
-                    "{} `{}` can't be a subteam of a subteam (`{}`)",
-                    team.kind(),
-                    team.name(),
-                    parent.name(),
-                );
-            }
-
```

Comment:
I don't 100% understand why this clause was left in #1046. @fmease Can you say why this is here?

Reply:
Sorry, I didn't get notified by the ping, maybe because this PR is marked as a draft?

Reply:
I kept this check because it's not clear where to place WGs and PGs that are children of subteams on the website. WGs and PGs are placed at the bottom of the page inside new distinct sections. I doesn't feel right to place WGs/PGs that aren't actually direct children of the team whose page we are looking at at the same level as other WGs/PGs especially since the user cannot draw any connection between the parent team and the groups.

Reply:
For example:

```
Team A
+-- Team A-A
|   +-- Working Group A-A-A
+-- Team A-B
```

The webpage of team A would look like:

```
Team A
--- Subteams
Team A-A
Team A-B
--- WGs
Working Group A-A-A
--- PGs
```

Now it's unclear if WG A-A-A is a direct child of A, A-A or A-B.